src/vmEntry.cpp

```diff
@@ -70,6 +70,14 @@ void VM::init(JavaVM* vm, bool attach) {
     PerfEvents::init();
 
     _asyncGetCallTrace = (AsyncGetCallTrace)dlsym(RTLD_DEFAULT, "AsyncGetCallTrace");
+    if (_asyncGetCallTrace == NULL) {
```

Comment:
Looks good, but I would leave a comment here when this is needed.
Something like in PR description would be sufficient.

Reply:
Added a comment.